src/Service.Host/client/src/components/PurchaseOrders/PurchaseOrderUtility.js

```diff
@@ -67,6 +68,31 @@ function PurchaseOrderUtility({ creating }) {
         }
     }, [orderNumber, reduxDispatch, creating]);
 
+    const columns = [
+        { field: 'id', headerName: 'Id', width: 100, hide: true },
+        { field: 'deliverySeq', headerName: 'Delivery', width: 100 },
+        { field: 'ourDeliveryQty', headerName: 'Qty', width: 100 },
+        {
+            field: 'dateRequested',
+            headerName: 'Request Date',
+            width: 100,
+            type: 'date'
+        },
+        {
+            field: 'dateAdvised',
+            headerName: 'Advised Date',
+            width: 100,
+            type: 'date'
+        },
+        {
+            field: 'availableAtSupplier',
+            headerName: 'Available at Supplier?',
+            width: 100,
+            type: 'singleSelect',
```

Comment:
you won't need to make this column a singleSelect column since the table isn't editable

Reply:
good point. updated the branch with these two fixes. appreciated!